flutter_local_notifications_linux/lib/src/dbus_wrapper.dart

```diff
@@ -47,5 +47,5 @@ class DBusWrapper {
 
   /// Creates a stream of signal with the given [name].
   DBusRemoteObjectSignalStream subscribeSignal(String name) =>
-      DBusRemoteObjectSignalStream(_object, _destination, name);
+      DBusRemoteObjectSignalStream(object: _object, interface: _destination, name: name);
```

Comment:
could you add a trailing comma to address the issue where it breaks 80 character line limit rule?